docs/source/polars-cloud/workspace/team.md

```diff
@@ -27,6 +27,7 @@ The invitee can use this link to join the workspace without requiring an email i
 <!-- dprint-ignore-start -->
 
 !!! info "Inviting existing users"
-Users that are already part of another workspace will get a prompt to join the workspace they are invited for.
+
+    Users that are already part of another workspace will get a prompt to join the workspace they are invited for.
```

Comment:
You don't get a prompt. If you're part of an org then workspace admins can add you willy-nilly.